src/dmd/mtype.d

```diff
@@ -4922,6 +4922,23 @@ extern (C++) final class TypeFunction : TypeNext
         return MATCH.nomatch;
     }
 
+    /**
+     * Check if the arguments of a call are possibly inverted and prints an
+     * additional error message if the call would compile.
+     */
+    extern (D) void checkInvertedMatch(Type tthis, Expressions* args, int flag = 0, const(char)** pMessage = null, Scope* sc = null)
+    {
+        if (args && args.length != 2)
+            return;
+        Expressions* invertedArgs = new Expressions(2);
+        (*invertedArgs)[0] = (*args)[1];
+        (*invertedArgs)[1] = (*args)[0];
+        if (const m = callMatch(tthis, invertedArgs, flag, null, sc))
+        {
+            errorSupplemental((*args)[1].loc, "would the parameters order be inverted ?");
```

Comment:
Would be nice if the error message contains a code snippet that the users can copy-paste.